src/any_agent/frameworks/agno.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, List, Optional
+from any_agent.config import AgentConfig, AgentFramework
+from any_agent.frameworks.any_agent import AnyAgent
+from any_agent.logging import logger
+from any_agent.tools.wrappers import import_and_wrap_tools
+
+try:
+    from agno.agent import Agent
+    from agno.models.litellm import LiteLLM
+
+    agno_available = True
+except ImportError:
+    agno_available = None
+
+
+class AgnoAgent(AnyAgent):
+    """Agno agent implementation that handles both loading and running."""
+
+    def __init__(
+        self, config: AgentConfig, managed_agents: Optional[list[AgentConfig]] = None
+    ):
+        if not agno_available:
+            raise ImportError(
+                "You need to `pip install 'any-agent[agno]'` to use this agent"
+            )
+        if managed_agents:
+            raise NotImplementedError(
+                "Managed agents are not yet supported in Agno agent."
+            )
+        self.config = config
+        self._agent = None
+        self._mcp_servers = None
+
+    def _get_model(self, agent_config: AgentConfig):
+        """Get the model configuration for an Agno agent."""
+        return LiteLLM(
+            id=agent_config.model_id,
+            **agent_config.model_args or {},
+        )
+
+    async def _load_agent(self) -> None:
+        if not self.config.tools:
```

Comment:
I fell into this trap in #70 .

If you don't put that then the main agent ignores the managed_agents and tries to solve it with the default tools.

```suggestion
        if not managed_agents and not self.config.tools:
```

Reply:
Currently that isn't necessary because of https://github.com/mozilla-ai/any-agent/pull/77/files#diff-8e36bbb2ae7abf556304d4ba4434155ef1ee52df9fe76e0ae9b25b5d96c9cbf8R26-R29 I think?

Basically, if you try to set managed_agents, I throw an exception in the constructor because Agno doesn't yet support multi_agent.

However, doesn't hurt to add it to make it easier for multi agent when that comes, so I'm fine with adding it in

Reply:
Correct, it is not needed, I just commented because it doesn't hurt and prevents making the same mistake I made